Ground_Engine/Create/AnalysisProperties.cs

```diff
@@ -0,0 +1,70 @@
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using BH.oM.Base.Attributes;
+using BH.oM.Geometry;
+using BH.oM.Geometry.CoordinateSystem;
+using BH.oM.Ground;
+using BH.Engine.Base;
+using BH.Engine.Geometry;
+
+namespace BH.Engine.Ground
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Creates an object containing the analysis properties for a contaminant sample.")]
+        [Input("totalOrDissolved", "Whether the specimen is total or dissolved (ERES_TORD).")]
+        [Input("accreditingBody", "Accrediting body and reference number (when appropriate) (ERES_CRED).")]
+        [Input("labName", "Name of testing labratory or organisation (ERES_LAB).")]
+        [Input("percentageRemoved", "Percentage of material removed (ERES_PERP). If no value is assigned, the default value will be 0.")]
+        [Input("sizeRemoved", "Size of material removed prior to test. Value represents lowest sized material removed (ERES_SIZE). If no value is assigned, the default value will be 0.")]
+        [Input("instrumentReference", "Instrument reference number or identifier (ERES_IREF).")]
+        [Input("leachateDate", "Leachate preperation date and time (ERES_LDTM). If no value is assigned, the default value will be 1/1/0001 12:00:00 AM.")]
+        [Input("leachateMethod", "Leachate preperation method (ERES_LMTH).")]
+        [Input("diluationFactor", "Dilution factor (ERES_DIL). If no value is assigned, the default value will be 0.")]
+        [Input("basis", "Basis (ERES_BAS).")]
+        [Input("location", "Analysis location (ERES_LOCN).")]
+        [Output("analysisProperties", "Properties related to the analysis undertaken on the contaminant.")]
+        public static AnalysisProperties AnalysisProperties(string totalOrDissolved = "", string accreditingBody = "", string labName = "", 
+            double percentageRemoved = 0, double sizeRemoved = 0, string instrumentReference = "", DateTime leachateDate = default(DateTime), string leachateMethod = "",
+            int diluationFactor = 0, string basis = "", string location = "")
+        {
+            return new AnalysisProperties() { TotalOrDissolved = totalOrDissolved, AccreditingBody = accreditingBody, LabName = labName, PercentageRemoved =percentageRemoved,
+            SizeRemoved = sizeRemoved, InstrumentReference = instrumentReference, LeachateDate = leachateDate, LeachateMethod = leachateMethod, DilutionFactor = diluationFactor, 
+            Basis = basis, Location = location};
+        }
```

Comment:
I am not sure I understand the point of those create methods as this doesn't add anything to the auto-generated component created from the object definition itself.

The only thing that would be required is move the default values directly on the properties themselves in the oM.
Which is generally a better solution anyways as it simplifies creating new objects from the code.